src/kernel/ship.test.c

```diff
@@ -570,11 +571,16 @@ static void test_ship_damage(CuTest *tc) {
     CuAssertIntEquals(tc, sh->size * DAMAGE_SCALE / 2, sh->damage);
     CuAssertIntEquals(tc, 50, ship_damage_percent(sh));
     sh->number = 2;
+    CuAssertIntEquals(tc, 50, ship_damage_percent(sh));
+    sh->number = 1;
+    scale_ship(sh, 2);
```

Comment:
Ich liebe ja nicht, dass hier scale_ship benutzt wird. Die Funktion ist eignetlich geschrieben, um aus einer Flotte ein oder mehrere Schiffe zu entfernen (wenn man die Flotte auftrennt). "Füge ein Schiff hinzu, dass den gleichen Schadensgrad wie die Flotte hat" war nie in der Beschreibung, auch wenn es das tut, Für die Lesbarkeit des Tests finde ich es einfacher, wenn direkt aud number, damage und size geschrieben wird.

Reply:
Andererseits ist ein Test, der sagt "wenn eine Flotte skaliert wird, bleibt ihr Schaden unverändert" nicht verkehrt. Und ich sehe, bind_ship.c erlaubt auch aus Lua, dass eine Flotte samt ihrem Schaden hochskaliert wird, wenn man in Lua ship.number setzt.